assets/stylesheets/_components.scss

```diff
@@ -142,6 +142,7 @@
 @import 'components/info-popover/style';
 @import 'components/input-chrono/style';
 @import 'components/keyed-suggestions/style';
+@import 'components/jetpack-colophon/style';
```

Comment:
Did you mean to insert above `keyed-suggestions`? Keeping these imports alphabetized makes for easier scanning.

Reply:
Didn't even notice. Will fix.